CenterForOpenScience/osf.io
# [#PLAT-696]Add preprints and registrations to user profile page

## Purpose
Update user's profile page by adding public registrations and preprints. Remove public components.


## Changes
Before change:
<img width="1426" alt="screen shot 2018-03-20 at 10 14 56 am" src="https://user-images.githubusercontent.com/4974056/37659925-9642b6e2-2c27-11e8-8f47-1c32133cf08d.png">
After change:
<img width="1305" alt="screen shot 2018-03-20 at 10 15 25 am" src="https://user-images.githubusercontent.com/4974056/37659951-a88100b6-2c27-11e8-8e1c-465ee3ea0799.png">
<img width="1369" alt="screen shot 2018-03-20 at 10 15 57 am" src="https://user-images.githubusercontent.com/4974056/37659970-b52dbf02-2c27-11e8-91cd-9d60f86466bb.png">


## QA Notes
1. checkout user's profile page.
things to check:

- If user has no preprints, preprints widget will still show up but saying "The user has no preprints"

- If user has no public registrations, registrations widget will still show up but saying "The user has no public registrations"

- If user has no quickfiles, quickfiles widget will disappear



## Side Effects



## Ticket
https://openscience.atlassian.net/browse/PLAT-696
